modules/standard/IO.chpl

```diff
@@ -5325,6 +5339,7 @@ proc _channel.readbits(ref v:integral, nbits:integral):bool throws {
    :arg v: a value containing *nbits* bits to write the least-significant bits
    :arg nbits: how many bits to write
 
+   :throws UnexpectedEofError
```

Comment:
For these writing methods, I'd add the additional description: "Thrown if the write operation exceeds the 'fileWriter's specified range"

Reply:
Makes sense. However should this say `channel` as opposed to `fileWriter`, since it's a method on `_channel`?

Reply:
I'd day `fileWriter` as `channel` is in the process of being deprecated and replaced with `fileWriter`/`fileReader`.